app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -23,7 +23,6 @@ def call
       trusted_proxies_list
     rescue => e
       raise Errors::Conjur::TrustedProxiesFetchFailed.new(
```

Comment:
don't we want to log the host id? can't it help the customer?

Reply:
i think no,
In most of the errors we already have this info, for example:
```
CONJ00039D Fetching trusted proxy list from host/conjur/settings/trusted_proxies
Authentication Error: #<Errors::Conjur::TrustedProxiesFetchFailed: CONJ00065E Failed to fetch trusted proxy list. Reason: '#<Errors::Authentication::Security::RoleNotFound: CONJ00007E 'cucumber:host:conjur/settings/trusted_proxies' not found>
```
So it will be strange to write it twice.

Also you can see it in the log flow, for example:
```
CONJ00039D Fetching trusted proxy list from host/conjur/settings/trusted_proxies
CONJ00040D 340 trusted proxies were fetched
CONJ00041D Deleting duplications from trusted proxy list
CONJ00042D Trusted proxy list contains 7 IP addresses
CONJ00043D Validating proxy list [1.1.1.1, 2.2.2.2]
Authentication Error: #<Errors::Conjur::InvalidProxy: CONJ00066E Invalid proxy '1.1.1.1'>

```